3d_armor/api.lua

```diff
@@ -465,8 +465,9 @@ end
 
 armor.load_armor_inventory = function(self, player)
 	local _, inv = self:get_valid_player(player, "[load_armor_inventory]")
+	local meta = player:get_meta()
```

Comment:
Move this line one line down.

Reply:
I've moved it one line down. Is it fixed now?

Reply:
Yes, it's correct.

Reply:
Okay. If you find anything else, please tell me. Thanks! 👍